server/src/shared/interfaces/response-objects/PaginatedUserBoards.ts

```diff
@@ -1,14 +1,24 @@
-import { SuperBoard } from '../../../schemas/board/super.board.schema';
+import { SchemaTimestampsConfig } from 'mongoose';
+import { UserPermission } from '../UserPermission';
 
-export interface PaginatedBoardsResponse {
-  boards: BoardWithoutSlidesAndPermissions[];
+export interface PaginatedBoardsResponseObject {
+  boards: PopulatedBoardResponseObject[];
   page: number;
   limit: number;
   order: string;
```

Comment:
Honestly, I'm not in favor of using the 'order' property just for specifying ascending/descending sorting. While we're not currently using it from the frontend, it would be much clearer to associate the 'order' property with the relevant column(s). I understand that it’s applied to the `updatedAt` field (someone could also assume it is related to createdAt), but if we introduce additional columns in the future, this won't be as intuitive.

For better clarity and flexibility, consider specifying the sort format directly, for example: `sort=some_column,asc` or `sort=-some_column`. This way, it’s easy to extend or modify, such as using `sort=some_column1,some_column2` for multi-column sorting. It’s a more scalable approach.


[Best Practices for Sorting in RESTful APIs](https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#sorting)

![image](https://github.com/user-attachments/assets/227e5c29-320e-4a56-9a50-2269045dc986)

Reply:
I can do that but in a separate ticket

Reply:
this approach is quite nice, i will use it for sure